src/UhdmAst.cpp

```diff
@@ -2492,11 +2492,28 @@ AstNode* visit_object(vpiHandle obj_h, UhdmShared& shared) {
             return new AstAttrOf(make_fileline(obj_h), AstAttrType::DIM_UNPK_DIMENSIONS,
                                  arguments[0]);
         } else if (objectName == "$bits") {
-            // If this is not an expression, explicitly mark it as data type ref.
-            // See exprOrDataType in verilog.y
-            AstNode* expr_datatype_p = arguments[0];
-            if (VN_IS(expr_datatype_p, ParseRef)) {
-                expr_datatype_p = new AstRefDType(make_fileline(obj_h), expr_datatype_p->name());
+            // Verilator's original parser allow $bits function to have 2 arguments
+            // but it is not defined in standard
+            if (arguments.size() != 1)
+                v3error("Number of arguments is not 1");
+            vpiHandle arg_itr = vpi_iterate(vpiArgument, obj_h);
```

Comment:
Please release the handles from `vpi_iterate`, `vpi_scan`, and `vpi_handle` using `vpi_release_handle`.

Reply:
Done